### DISCUSSION TITLE
Importing images into webpack and VSCode

### DISCUSSION THREAD

**Discussion by cmeffley** [nss-evening-cohort-14/evening-client-side#158] [Help Me!] [answered]
Chris Meffley

No other tickets

Having difficulties with .png pictures in webpack. I need a refresher on importing them into VS Code, and once they're there, getting them linked correctly in the code.

No errors

https://stackoverflow.com/questions/42961712/how-to-include-image-as-markdown-in-visual-studio-code
https://stackoverflow.com/questions/38682966/visual-studio-images-not-displaying#:~:text=You%20need%20to%20add%20an,Resources.
https://code.visualstudio.com/docs/getstarted/tips-and-tricks
https://code.visualstudio.com/docs/getstarted/userinterface

I've drug the picture into the VS code file by way of folders on my local machine, but not sure if there is an easier way to do it. Once the picture is in my VS code, finding the path completely eludes me. I tried import example from ' ' as well as just trying to type the path in where I want it to go, but nothing seems to be working.

https://github.com/cmeffley/cm-personal-bio-site
https://github.com/cmeffley/cm-personal-bio-site/tree/main/src

**Comment by drteresavasquez** [comment]
And you've reviewed the webpack instructions as well, right? https://github.com/nss-evening-cohort-14/webpack-template#including-images-with-webpack

**Comment by drteresavasquez** [comment]
Also, another suggestion is to use Firebase to host your images. If you go into one of your projects, you can click on storage in the navigation. You can upload your images there and then use the links in your project. Don't worry about the ? on this image but the circled part is the url you can use. If you click it, it will copy. 

![image](https://user-images.githubusercontent.com/29741570/110224440-2fb29800-7ea1-11eb-928e-42e2f87b2e74.png)

**Comment by cmeffley** [comment, marked as answer]
Thank you.... so my problem was that I wasn't tracking to the img folder correctly. I kept only doing: `/`, `./`, `../`  thinking that should get me to the correct folder, but ultimately my import was: `import projectCards from '../../../images/productCards.png';`. I found Tad's module import discussion and figured it out.  Then I had to take the quotations off the projectCards in the object. 
Thank you for the FireBase tip. I will look into that. Getting the picture into VS Code seems to give me the most trouble for some reason.

**Reply by drteresavasquez** [reply]
Yeah. Copying it over is just like saving an image to your computer. You could also find the directory and add it that way. Glad @tsekeres ticket helped!! ➕➕Tad!
